Gemfile.lock

```diff
@@ -73,17 +73,17 @@ GEM
     thread_safe (0.3.6)
     tzinfo (1.2.5)
       thread_safe (~> 0.1)
-    zxcvbn-ruby (0.0.2)
+    zxcvbn-ruby (0.1.2)
```

Comment:
### [zxcvbn-ruby](http://github.com/envato/zxcvbn-ruby)

**Minor** version upgrade :chart_with_upwards_trend::large_orange_diamond: 0.0.2 → 0.1.2

[_~~change-log~~, ~~source-code~~_]





<details>
<summary>Commits</summary>

A change of **53** commits. See the full changes on [the compare page](https://github.com/envato/zxcvbn-ruby/compare/v0.0.2...v0.1.2).

These are the first 10 commits:
- (a53fc31) [Fix yaml type coercion](https://github.com/envato/zxcvbn-ruby/commit/a53fc31f99fe3524d5c40219efb0aeb4a14b51be)
- (50caca3) [Merge pull request #8 from sj26/fix-yaml-type-coercion](https://github.com/envato/zxcvbn-ruby/commit/50caca3612c48c8719e5b28b5367c1f4f0cbbf05)
- (86db966) [Use Ruby Enumerable methods](https://github.com/envato/zxcvbn-ruby/commit/86db9668cd8bd848ce0a599a0dadefdb96acbbec)
- (3d01c23) [Reformatted case](https://github.com/envato/zxcvbn-ruby/commit/3d01c238a16a5115c4610865ecf4fc7cc481559c)
- (72d1900) [Simplified to_hash](https://github.com/envato/zxcvbn-ruby/commit/72d1900a66d888b12ac0018f2c87a0725e9df28c)
- (c845cef) [Removed ruby’s own min method](https://github.com/envato/zxcvbn-ruby/commit/c845cefbd7ef10df2a5316c809e4cecdd6edaf1e)
- (a1a0b68) [Simplified averaging method](https://github.com/envato/zxcvbn-ruby/commit/a1a0b6855c7a94e06adefcee16b5311d2f7d25d0)
- (811f2a6) [Removed modifying from re_match_all](https://github.com/envato/zxcvbn-ruby/commit/811f2a6d30d5dd050102cfb22ff488c1a2a1ba3a)
- (1e69dcf) [Use map to create an array, instead of each](https://github.com/envato/zxcvbn-ruby/commit/1e69dcf842c64a37e1fe878f5d0cd564291123cd)
- (f256be9) [Simplified method](https://github.com/envato/zxcvbn-ruby/commit/f256be9334c3ecd81ddfd0cf93073d7a1168e6da)


</details>